github/workflows/main.yml

```diff
@@ -0,0 +1,26 @@
+name: Upcloud go api test
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
Hmmm... Unsure if the CI/CD runs against PR:s, since at least this PR is not checking anything. My TF one is running the GitHub actions.

Reply:
resolved by fixing the path 😁